atom/browser/autofill/atom_autofill_client.cc

```diff
@@ -124,6 +124,10 @@ ukm::UkmRecorder* AtomAutofillClient::GetUkmRecorder() {
   return ukm::UkmRecorder::Get();
 }
 
+AddressNormalizer* AtomAutofillClient::GetAddressNormalizer() {
+  return nullptr;
```

Comment:
cc @darkdh

Reply:
This will enhanced fields filling accuracy. I will do it as follow up